src/components/ui/sidebar/app-sidebar.tsx

```diff
@@ -126,7 +126,13 @@ export function AppSidebar({
       </SidebarContent>
 
       <SidebarFooter>
-        {(facilitySidebar || selectedOrganization) && <FacilityNavUser />}
+        {(facilitySidebar || selectedOrganization) && (
+          <FacilityNavUser
+            {...(facilitySidebar && {
+              selectedFacilityId: selectedFacility?.id,
+            })}
+          />
```

Comment:
correct me if i'm wrong, this could be made easier right?

```suggestion
          <FacilityNavUser selectedFacilityId={facilitySidebar ? selectedFacility?.id : undefined} />
```

Reply:
Technically selectedFacility?.id should be undefined when an org is selected, but let's do this just to be safe 😆